Lib9c/SerializeKeys.cs

```diff
@@ -152,6 +152,7 @@ public static class SerializeKeys
         public const string TradableIdKey = "ti";
         public const string OrderReceiptListKey = "orl";
         public const string BuyerAgentAddressKey = "bga";
+        public const string HackAndSlashBuffStateKey = "has_buff";
```

Comment:
this value use to derive address. i think it would be better move te `Addresses` or AvatarState extension method.